### PR TITLE
Api endpoints

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/controllers/cats_controller.rb [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+  end
+  
+def destroy
 end
+    # Handle strong parameters, so we are secure
+    private
+    def cat_params
+        params.require(:cat).permit(:name, :age, :enjoys, :image)
+    end
+
```
Comment (sjproctor) [inline app/controllers/cats_controller.rb]: This file needs some love on the indentation

File: app/controllers/cats_controller.rb [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
+    cat = Cat.find(params[:id])
+    cat.update(cat_params)
+    if cat.valid?
+      render json: cat
+    else
+      render json: cat.errors
+    end
+  end
+  
+def destroy
```
Comment (sjproctor) [inline app/controllers/cats_controller.rb]: If you don't have specs for update and destroy, you should remove the controller methods.

**Review by sjproctor — COMMENTED** [review]
just a couple things.

**Comment by ghost** [comment]
Made necessary corrections.

File: app/controllers/cats_controller.rb [hunk 3]
```diff
@@ -1,30 +1,18 @@
...
+    cat = Cat.create(cat_params)
+    render json: cat
 end
 
-#   something
\ No newline at end of file
+private
+def cat_params
+    params.require(:cat).permit(:name, :age, :enjoys, :image)
+end
```
Comment (sjproctor) [inline app/controllers/cats_controller.rb]: lines 3 through 17 should be indented one. That way it is nested inside the class.

**Review by sjproctor — COMMENTED** [review]
one more thing

**Review by sjproctor — APPROVED** [review]

**Pull request merged**